data/hai missions.txt

```diff
@@ -435,3 +435,585 @@ mission "Unfettered returning home"
+	npc save
+		personality escort
+		government "Merchant"
+		fleet
+			names "civilian"
+			variant
+				"Bulk Freighter (Hai)" 3
+	
+	npc
+		personality staying
+		government "Pirate"
+		system "Ultima Thule"
+		fleet "Small Northern Pirate"
+	npc
+		personality staying
+		government "Pirate"
+		system "Rajak"
+		fleet "Large Northern Pirate"
+	npc
+		personality staying
+		government "Pirate"
+		system destination
+		fleet "Large Northern Pirate"
+		
+	on complete
+		dialog `The Bulk Freighters are loaded with supplies without any problem. When one of the spaceport workers asks Turner where the supplies are going, he replies, "These are just to expand a business venture of mine on, uh, Glory."`
+
+mission "Expanding Business [7]"
+	landing
+	name "A New Proposition"
+	description "Escort the freighters back to <destination> to begin construction on the shipyard."
+	destination "Greenwater"
+	to offer
+		has "Expanding Business [6]: done"
+	
+	npc
+		personality staying
+		government "Pirate"
+		system "Cardax"
+		fleet "Small Northern Pirate"
+	npc
+		personality staying
+		government "Pirate"
+		system "Rajak"
+		fleet "Large Northern Pirate"
+		
+	on complete
+		event "shipyard on greenwater" 180
+		conversation
+			`	Construction on the shipyard begins almost immediately after the supplies are unloaded from the freighters, starting with the demolition of the rest of the park. A downtrodden looking crowd gathers to watch the demolition, but Turner ensures to you that there are plenty of other parks in the city for them to go to.`
+			`	"Something big is going to come of this, Captain <last>," Turner says. "Just know that you were all part of it. You'll know the shipyard is done when you find an extra nine thousand credits in your pocket each day. Goodbye until then, Captain <last>."`
+
+event "shipyard on greenwater"
+	planet "Greenwater"
+		shipyard "Basic Ships"
+		shipyard "Syndicate Basics"
+		shipyard "Megaparsec Basics"
+		shipyard "Megaparsec Advanced"
+		outfitter "Common Outfits"
+		outfitter "Syndicate Advanced"
+		outfitter "Lovelace Basics"
+	fleet "Small Human Merchants (Hai)"
+		add variant 20
+			"Sparrow"
+		add variant 20
+			"Star Barge"
+		add variant 20
+			"Shuttle"
+		add variant 5
+			"Freighter"
+			"Wasp" 2
+		add variant 10
+			"Freighter"
+			"Shuttle" 2
+			"Quicksilver"
+		add variant 15
+			"Bounder"
+			"Wasp" 2
+		add variant 5
+			"Manta"
+		add variant 5
+			"Freighter" 2
+			"Manta"
+	fleet "Large Human Merchants (Hai)"
+		add variant 6
+			"Freighter (Proton)" 2
+			"Quicksilver" 2
+		add variant 6
+			"Freighter" 4
+			"Quicksilver" 2
+			"Splinter" 1
+		add variant 3
+			"Freighter" 4
+			"Quicksilver" 2
+			"Splinter (Laser)" 1
+		add variant 3
+			"Freighter" 4
+			"Quicksilver" 2
+			"Splinter (Proton)" 1
+
+mission "Expanding Business: shipyard complete"
+	landing
+	invisible
+	to offer
+		has "event: shipyard on greenwater"
+	on offer
+		"salary: Turner Incorporated" = 14000
```

Comment:
I feel like this salary increase of 14000 is too high.  Tribute from Earth is nearly equivalent and a salary from the FW storyline is much less than this.  

Both missions either need their rewards reduced by nearly a factor of 10, or the difficulty or skin in the game ramped up.  For example having to invest in the business giving you a higher stake in the business or buying materials yourself and delivering them to the system (not sure if this mechanic would work).

Reply:
Given the nature of conquering planets (i.e. it's really just something extra for players looking to do something else and isn't exactly all that supported), I wouldn't use the tribute value of a planet to compare to a salary. 

As for the FW storyline, the Free Worlds aren't exactly full of individuals doing it just for the credits, but Turner is all about how much money is involved. (Notably, he talks about your salary in terms of how many millions per year it nets you, not how much per day.) The main reason for the salary in the FW campaign is to cover most of your crew costs for the type of ship you're expected to have at that time in the story. These missions take place after the FW campaign, though. Seeing as how the Wanderer campaign gives no salary so far, I set the Turner Incorporated salary high compared to the FW salary in order to help players going into the Wanderer campaign. It's also important to note that this 14,000 salary is only obtainable over one year after completing the first half of this storyline, which itself is only obtainable right after finishing the FW campaign (which opens up the Wanderer campaign). Given the time difference between getting a salary of 5,000 to 14,000, I would expect the player to have finished quite a bit of the Wanderer campaign by then, at which point they may need another boost in their salary. While the Wanderer campaign is beatable in one ship that the FW salary may mostly cover, [not everyone may have the easiest of time accomplishing that](https://github.com/endless-sky/endless-sky/issues/2054), in which case an additional salary is either a huge help or simply a nice bonus.

I may consider dropping the salaries to 4k and 9k, though.

Reply:
I feel like any salary at all may be a bit much for these missions, as currently written. The only justification is:

> the profits are sure to be phenomenal, profits that you would be sharing in by being a founding member of this outfitter.

What exactly are you being paid to do? Not much beyond a few simple escort jobs, and nothing at all on an ongoing basis. Someone with the personality that's depicted here would be loath to throw away 2-5 million credits a year paying a business partner to do... nothing. 

I don't see any reason why Turner would willingly pay you so much for so little. Partnership businesses typically involve both partners doing equal work--or at least more than nothing. No partnership would survive one of the partners being paid 2-5M/year to do nothing.

Maybe the player should negotiate a license fee for the use of his/her name in conjunction with the enterprise (not to mention the protection work), rather than Turner just offering to pay you an enormous amount of money forever for doing practically nothing.

Even if you change that, I would approve of reducing the daily salaries a *lot*. 10k/day is enough to make a daily profit while flying around in a Hurricane, not even counting your Free Worlds pension.

Reply:
It's branding. Turner uses your name and the fact that you helped set up the shipyard and outfitter in order to advertise it. The money you get is royalties for that. The justification is more in the line
>with a name like yours backing up this business

than in 
>being a founding member of this outfitter.

And given that the cost of *a single ship or outfit* can exceed 2 to 5 million credits (not the same as the profit the business earns from selling the ship or outfit, I know, but we don't exactly have any estimates on the profit margins*), I'm sure Turner is fine with the exchange of being able to use the player's name in advertising, even if the player contributes nothing to the business after helping to set it up. Just think about the fact that Turner is probably making several hundred times more credits from owning the business than he is giving you, especially given the fact that the Turner shipyard and outfitter in Hai space has a monopoly on lower cost ships.

I'll substantially decrease the Turner Incorporated salary to 2k at first and 4k in the end.

\* I decided to try and do some research on the profit margins for selling cars (because why not?), and some random people on the internet who claim to be car dealers say that profit margins can range form 1% to 4% on the low end up to 10% to 15% for more expensive and luxurious cars. Assuming the numbers are the same for selling a spaceship, Turner's shipyard is probably more on the low end for profit margins, but I'm sure that's more than made up for by the fact that it's the only shipyard selling ships cheaper than 3 million credits in all of Hai space, which we know has billions of Hai and presumably millions of humans to potentially sell it. Do with that knowledge what you wish. The more you know.

Reply:
If the payment comes from using your name (which makes sense), I'd like that to be more obvious, or have the player negotiate it, or make it more explicit that there's a legal agreement governing the use of the name in exchange for a fee.

Reply:
Any suggestions? I'm averse to having a negotiation occur because I feel like being able to negotiate would entail having to possibly have multiple different salaries that the player can end up negotiating for, which adds unneeded complexity to the mission in my eyes. Having a conversation explicitly state the legality of the transaction you're about to partake in also sounds awkward to me.

How about instead of referring to you having a share in the profit as being a founding member, he says
>the profits are sure to be phenomenal, profits that you would be sharing in by having your named attached to this outfitter.

I feel that that's explicit enough in stating that the reason you're paid is because your name is being used.